src/pages/accounts/components/NewAccountDialog.tsx

```diff
@@ -1,19 +1,39 @@
 import React from "react";
 import { Grid, Chip, Typography, TextField, MenuItem } from "@mui/material";
+import { requestNewAccount } from "../AdminAccountsPage.slice";
+import { useAppDispatch } from "../../../app/common/hooks";
 import { INewAccount, rolesEnum } from "../types";
 import Dialog from "../../../shared/GenericDialog";
 
+function isBlank(str: string) {
```

Comment:
I think we have multiple of these functions. Should we put this in a shared file?

Reply:
Imported from where it existed before

Reply:
Added file for shared functions